packages/perseus/src/util/interactive.ts

```diff
@@ -2591,7 +2591,7 @@ _.extend(GraphUtils.Graphie.prototype, {
         // and maximum angle values bounding a circle sector, and
         // the third column refers to the cursor name that will be
         // applied if the mouse position falls inside the given sector.
-        const cursorAxes = [
+        const cursorAxes: Array<[number, number, string]> = [
```

Comment:
This was necessary because we do some `<` and `>` below on the first two elements in each 3-tuple, but TS was inferring the 3-tuple as `Array<number | string>` and was complaining about `<` and `>` being used with strings.